Multisig/UI/App/Unlock/FaceIDUnlockViewController.swift

```diff
@@ -19,7 +19,7 @@ class FaceIDUnlockViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        label.setStyle(.body)
+        label.setStyle(.title1.weight(.semibold))
```

Comment:
Can you please define new text style? I would avoid using inline customized styles because it caused me a lot of pain during rebranding, and I believe in future we will use it more frequent